kbs/config/kubernetes/base/kustomization.yaml

```diff
@@ -4,16 +4,22 @@ namespace: coco-tenant
 
 images:
 - name: kbs-container-image
-  newName: ghcr.io/confidential-containers/key-broker-service
-  newTag: built-in-as-v0.8.2
+  newName: ghcr.io/confidential-containers/staged-images/kbs-grpc-as
+  newTag: dc01f454264fb4350e5f69eba05683a9a1882c41
+- name: as-container-image
+  newName: ghcr.io/confidential-containers/staged-images/coco-as-grpc
+  newTag: f76cf9272a0bc0be45864f4b49ba857793fd52f0
```

Comment:
What we have discussed previously regarding these images is that the kubernetes deployment is intended for end users and such will always be pinned to the most recent release image.

The docker compose file on the other hand will always pull the latest staging images. This is for developers.

This helps simplify the release a bit. Note if you add a new overlay that depends on the release version, that you should also update the release doc and the new release script to make sure it is updated.

Reply:
I could not find the release images for AS but looking at it again I can see they are available after all. I'll get this fixed tomorrow and see if I can get moved to an overlay nicely.

Reply:
For some reason the naming convention for the release images and staged images is different.

Reply:
https://github.com/confidential-containers/trustee/issues/377

Reply:
I'm just noticing what the issue is: I first checked "packages from trustee repo" only. If I look at all the packages/images published by confidential-containers org, I can see attestation-service with 0.8.2 tags.

I believe we are not publishing attestation-services images from trustee repo, which probably needs fixing?

Reply:
Ok nevermind, just saw the issue. Thanks!